gradle/dependencies.gradle

```diff
@@ -2,19 +2,11 @@ dependencies {
     implementation 'com.google.guava:guava:31.1-jre'
 
     // Use JUnit test framework
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
-    testImplementation 'org.mockito:mockito-inline:4.5.1'
-    compileOnly 'com.github.spotbugs:spotbugs-annotations:4.7.0'
-    
-    // Core Http library 
-    implementation 'com.microsoft.graph:microsoft-graph-core:3.0.0-SNAPSHOT'
-    
-    implementation 'com.microsoft.kiota:microsoft-kiota-abstractions:0.0.1-SNAPSHOT'
-    implementation 'com.microsoft.kiota:microsoft-kiota-abstractions:0.0.1-SNAPSHOT'
-    implementation 'com.microsoft.kiota:microsoft-kiota-authentication-azure:0.0.1-SNAPSHOT'
-    implementation 'com.microsoft.kiota:microsoft-kiota-http-okHttp:0.0.1-SNAPSHOT'
-    implementation 'com.microsoft.kiota:microsoft-kiota-serialization-json:0.0.1-SNAPSHOT'
-    implementation 'com.microsoft.kiota:microsoft-kiota-serialization-text:0.0.1-SNAPSHOT'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.1'
+    testImplementation 'org.mockito:mockito-inline:4.8.1'
 
-}
+    api 'com.google.code.gson:gson:2.10'
```

Comment:
I don't think we need to expose Gson here. And we might be missing some kiota dependencies, please cross check with core